### PR TITLE
Feature/mobile review

### PR DESCRIPTION
**Pull request by pedrohw12**

### CODE REVIEW

File: components/common/CardCreateOrder/styles.tsx [hunk 1]
```diff
@@ -32,6 +32,18 @@ export const ImageWithBackground = styled.div<ImageProps>(
 `
 )
 
+export const ImageContainer = styled.div`
+  height: 520px;
+  margin: auto;
+  .card-img {
+    width: auto !important;
```
Comment (juo-dev) [inline components/common/CardCreateOrder/styles.tsx:39]: Without 'important' doesn't work?

**Review by karacurt — APPROVED** [review]
💯

File: pages/marketplace/index.tsx [hunk 2]
```diff
@@ -19,14 +19,19 @@ import {
...
+  MobileVersionMessage
 } from '../../styles/marketplace/styles'
 
 const MarketplaceHome: NextPage = () => {
   const { reconnect } = useContext(Web3Context)
   const { orders } = useContext(MarketplaceContext)
+  const [isMobile, setIsMobile] = useState(false)
 
   useEffect(() => {
+    if ('ontouchstart' in document.documentElement && navigator.userAgent.match(/Mobi/)) {
```
Comment (juo-dev) [inline pages/marketplace/index.tsx:32]: You know react-device-detect, isn't it a better option (doubt)?

**Review by juo-dev — APPROVED** [review]
Nice!! 🚀

File: components/common/CardCreateOrder/styles.tsx (hunk 1 above)
Comment (pedrohw12) [reply components/common/CardCreateOrder/styles.tsx:39]: As far as I remember, I just got like this. Don't know the use for this, but works fine.

File: pages/marketplace/index.tsx (hunk 2 above)
Comment (pedrohw12) [reply pages/marketplace/index.tsx:32]: Didn't know. I will search for this. Thanks !

Comment (pedrohw12) [reply pages/marketplace/index.tsx:32]: This lib is using the same as this code (based on user agent). Maybe this code works fine.

**Pull request merged**